docs/docs/documentation/getting-started/quick-start.md

```diff
@@ -9,16 +9,53 @@ sidebar_position: 30
 Looking to get started with plugins? For more information, see the [Plugins](../concepts/plugins.md).
 :::
 
-To get started with Solace Agent Mesh, you must first create a project.
+To get started with Solace Agent Mesh, you can either create a complete project or use the preset configuration from the Docker image.
```

Comment:
Change this sentence to introduce the quick start in the actual order the subsections appear. Something like:

"To get started with Solace Agent Mesh, you can either use the preset configuration from the Docker image or create your own complete project."